src/main/resources/libgamerule.accesswidener

```diff
@@ -0,0 +1,4 @@
+accessWidener	v1	named
```

Comment:
Not sure what this file is used for but seems to be just docs, should be removed

Reply:
This isn't docs, this is essential. Access Wideners mean we don't need to dance around private classes.